### ISSUE TITLE
Running example in readme.md returns error

### ISSUE THREAD

**Issue by 4psireal2** [hejops/reaction-smiles#1]
Running this command line in python gave me an error

```python
python3 smartjoin.py 'C1CS[C+](S1)C1=CC=CC=C1.CCCC[Sn](CCCC)(CCCC)CC=C'
Traceback (most recent call last):
  File "smartjoin.py", line 19, in <module>
    from transforms import *
  File "/home/jung42/qod/science/reaction_prediction/playground/reaction-smiles/transforms.py", line 226, in <module>
    transformations = important_transformations | transformations | ugly_transformations
```

**Comment by hejops** [comment]
The `dict | dict` syntax requires python 3.9 or later. I'll push a fix that uses the older syntax in a bit.

**Comment by hejops** [comment]
Fixed in d4f43d84de1a464ee9d67a1843fcd4b7979cf970
